security-setup

```diff
@@ -403,10 +413,13 @@ class Zookeeper(Component):
 
     def consul_ssl(self):
         "turn on consul ssl"
-        with self.modify_security() as config:
-             config.setdefault('zk_consul_ssl', 'true')
-             config.setdefault('zk_consul_ssl_verify', 'false')
-             print('configuring consul ssl defaults')
+        if not self.args.no_ssl:
+            with self.modify_security() as config:
+                 config.setdefault('zk_consul_ssl', 'true')
+                 config.setdefault('zk_consul_ssl_verify', 'false')
+                 print('configuring consul ssl defaults')
+        else:
+            print('disable consul ssl config')
```

Comment:
this doesn't actually disable Consul SSL. You need to set those flags to the correct values. This script needs to be able to be run multiple times to update security preferences.